cmd/vulcan-exposed-rdp/main.go

```diff
@@ -121,12 +122,26 @@ func isExposedRDP(host, port string, timeout int) []report.Vulnerability {
 
 	if CCPDURegex.Match(b) && b[5] == byte('\xd0') {
 		exposedRDP.Details += fmt.Sprintf("RDP host detected in port: %s", port)
+		exposedRDP.AffectedResource = fmt.Sprintf("%s", port)
```

Comment:
Should we add the protocol too? Just to make it clearer for the user.

Reply:
Agree, I'll add it to the affected resource for more clarity.